cmd/blob-server/main.go

```diff
@@ -156,6 +158,49 @@ func main() {
 		})
 	})
 
+	router.GET("/avail/verify/:blockHeight/:extIndex", func(c *gin.Context) {
+		blockHeight := c.Param("blockHeight")
+		extIndex := c.Param("extIndex")
+		blockHeightUint, err := strconv.ParseUint(blockHeight, 10, 64)
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"message": fmt.Sprintf("blockHeight %s is not a valid uint64", blockHeight),
+			})
+			return
+		}
+		extIndexUint, err := strconv.ParseUint(extIndex, 10, 64)
+		if err != nil {
+			c.JSON(http.StatusBadRequest, gin.H{
+				"message": fmt.Sprintf("extIndex %s is not a valid uint64", extIndex),
+			})
+			return
+		}
+		verifier, err := availVerifier.NewDAVerifier(
+			"./avail-config.json",
+			ethEndpoint,
+			"0x967F7DdC4ec508462231849AE81eeaa68Ad01389", // Avail bridge deployed on Sepolia
+			"0x6B26173C8afF316919542df8dA5A57888e398ee1", // Custom Vector verifier contract deployed on Sepolia
+		)
+		if err != nil {
+			c.JSON(http.StatusInternalServerError, gin.H{
+				"message": fmt.Sprintf("failed to create verifier: %v", err),
+			})
+			return
+		}
+		success, err := verifier.VerifyDataIncluded(blockHeightUint, extIndexUint)
+		if !success || err != nil {
+			c.JSON(http.StatusOK, gin.H{
```

Comment:
proly
```suggestion
			c.JSON(http.StatusInternalServerError, gin.H{
```

Reply:
Ah no it isn't a server issue, verification failed on chain

Reply:
can we separate out `!success` and `err != nil` in that case?